documentation/docs/Contributing/Contributing_to_repository/development-pipeline.md

```diff
@@ -3,11 +3,11 @@ sidebar_position: 1
 ---
 # The Development Pipeline
```

Comment:
@sof202 The word pipeline in this context could get confused with data processing pipeline. I wonder if a title for this page like - "Contributions - A Quick Start Guide" might be more appropriate?

Reply:
I agree. This is a better name for it.